trajectory_msgs/JointTrajectory[] jointTrajArray
